### DISCUSSION TITLE
status data removed in streams.json

### DISCUSSION THREAD

**Discussion by Dukvah** [iptv-org/community#402] [Q&A] [answered]
normally i used to get channel data from [this](https://iptv-org.github.io/api/streams.json) link but today i saw that "status" key has been removed. So I got an error. Is there something I can use as an alternative?

The json file in the link used to look like this:

[
  //...
  {
    "channel": "BBCNews.uk",
    "url": "http://1111296894.rsc.cdn77.org/LS-ATL-54548-6/index.m3u8",
    "http_referrer": "http://example.com/",
    "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64)",
    "status": "online",
    "width": 1280,
    "height": 720,
    "bitrate": 565040,
    "frame_rate": 29.97,
    "added_at": "2022-07-13T04:41:45Z",
    "updated_at": "2022-07-13T04:41:45Z",
    "checked_at": "2022-08-13T14:05:19Z"
  }
  //...
]

**Comment by freearhey** [comment, marked as answer]
Unfortunately, we can no longer update the status of the stream, so it has been removed from the JSON output in https://github.com/iptv-org/iptv/pull/12628.

You can read more about the causes of these changes here: https://github.com/orgs/iptv-org/discussions/12#discussioncomment-5219050
